adapters/inmobi/inmobi.go

```diff
@@ -124,6 +124,9 @@ func getMediaTypeForImp(impId string, imps []openrtb2.Imp) openrtb_ext.BidType {
 			if imp.Video != nil {
 				mediaType = openrtb_ext.BidTypeVideo
 			}
+			if imp.Native != nil {
```

Comment:
Just a call out since this is logic was already present in your adapter code: Since your adapter is determining the bid type based on what object of the imp is set i.e `banner`, `video` or `native`, this function might return the wrong media type if the imp was a multi-format imp i.e an imp that has both `banner` and `video` set, for example.

Reply:
This should be fine. Will likely look into this in the future though (and stop relying on the bid request passthrough.)